testdata/Pipfile.lock

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
My understanding is that the lockfile should always be checked in to the repo, so that the next person that tries to install dependencies will be bale to use the same versions as the ones that were tested during development.